doc/src/documentation-style-guide.rst

```diff
@@ -0,0 +1,1278 @@
+SymPy in the Windows file system, you can prepend ``cd /mnt/`` to your file
+path in Windows, and run in your shell to navigate to the folder. (Also note
+that Linux uses ``/`` instead of ``\`` for file paths.)
+
+This method provides better compatibility than Cygwin or MSYS2 and more
+convenience than a virtual machine if you partially need a Linux environment
+for your workflow, however this method is only viable for Windows 10 64-bit
+users.
+
+2. Build the Documentation
+--------------------------
+
+The documentation can be built by running the ``makefile`` in the ``doc``
+subdirectory.
+
+To start, in your preferred web browser, use the drop down menu and select
+“open file” to navigate into the sympy/doc folder saved on your computer. In
+the doc folder, select the _build folder, then the html folder, and in the html
+folder, open the index.html file.
+
+To build the HTML documentation, run::
+
+   cd doc
+
+   make html
+
+This builds a local version of the documentation in ``doc/_build/html`` in your
+web browser.
+
+Open ``_build/html/index.html``.
+
+3. Make a Contribution
+----------------------
+
+For in-depth instructions on how to contribute to SymPy’s code base including
+coding conventions, creating your environment, picking an issue to fix, and
+opening a pull request, please read our full `Development Workflow
+<https://github.com/sympy/sympy/wiki/Development-workflow>`_ guide.
+
+Narrative Documentation Guidelines
+==================================
+
+Extensive documentation, or documentation that is not centered around an API
+reference, should be written as a narrative document in the Sphinx docs (located
+in the `doc/src directory
+<https://github.com/sympy/sympy/tree/master/doc/src>`_). The narrative documents
+do not reside in the Python source files, but as standalone restructured files
+in the doc/src directory. SymPy’s narrative documentation is defined as the
+collective documents, tutorials, and guides that teach users how to use SymPy.
+Reference documentation should go in the docstrings and be pulled into the RST
+with autodoc. The RST itself should only have narrative style documentation
+that is not a reference for a single specific function.
+
+Best Practices for Writing Documentation
+----------------------------------------
+
+Please follow these formatting, style, and tone preferences when writing
+documentation.
+
+Formatting Preferences
+^^^^^^^^^^^^^^^^^^^^^^
+
+In order for math and code to render correctly on the SymPy website, please
+follow these formatting guidelines.
+
+Math
+~~~~
+
+Text that is surrounded by dollar signs $ _ $ will be rendered as LaTeX math.
+Any text that is meant to appear as LaTeX math should be written as ``$math$``.
+In the HTML version of the docs, MathJax will render the math.
+
+**Example**
+
+::
+
+    The Bessel $J$ function of order $\nu$ is defined to be the function
+    satisfying Bessel’s differential equation.
+
+LaTeX Recommendations
+~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
@lglattly it occurs to me that we should put more here for people who are new to LaTeX. Here are some good resources for people to learn how to write math in LaTeX

- https://math.meta.stackexchange.com/questions/5020/mathjax-basic-tutorial-and-quick-reference
- https://en.wikibooks.org/wiki/LaTeX/Mathematics
- https://www.overleaf.com/learn/latex/Mathematical_expressions

And some basic tips for people new to LaTeX:

- If a docstring has any latex, make sure to make it "raw" [this is also mentioned below somewhere, Maybe you can cross reference it with a rst internal link]
- If you aren't sure how to render something, you can use the SymPy `latex()` function [cross reference this like :func:\`~.latex\`]. But be sure to srip out the unimportant parts (the bullet points below).
- Always check the final rendering to make sure it looks the way you expect it to.

Reply:
Added these notes, thanks!